visualpic/data_handling/data_container.py

```diff
@@ -157,6 +157,21 @@ def get_species(self, species_name):
         raise ValueError("Species '{}' not found. ".format(species_name) +
                          "Available species are {}.".format(available_species))
 
+    def add_derived_field(self, derived_field):
+        """Adds a derived field."""
```

Comment:
Could you add here a `Parameters` section that describes how `derived_field` should look like?

Reply:
Good point. I added something. Please check if it is OK.